cmd/payload-testing-prow-plugin/server.go

```diff
@@ -182,12 +182,27 @@ const (
 )
 
 func (s *server) handleIssueComment(l *logrus.Entry, ic github.IssueCommentEvent) {
-	if comment := s.handle(l, ic); comment != "" {
-		s.createComment(ic, comment, l)
+	if comment, additionalPRs := s.handle(l, ic); comment != "" {
+		org := ic.Repo.Owner.Login
+		repo := ic.Repo.Name
+		number := ic.Issue.Number
+		user := ic.Comment.User.Name
+		s.createComment(org, repo, number, comment, user, l)
+		originalPRRef := fmt.Sprintf("%s/%s#%d", org, repo, number)
+		for _, pr := range additionalPRs {
+			o, r, n, err := pr.GetOrgRepoAndNumber()
+			if err != nil {
```

Comment:
When `err != nil` then `o, r, n` are usable. I would also `continue` to the next iteration other than logging